lib/matplotlib/__init__.py

```diff
@@ -597,7 +597,14 @@ def gen_candidates():
 _all_deprecated = {*_deprecated_map, *_deprecated_ignore_map}
 
 
-@docstring.Substitution("\n".join(map("- {}".format, rcsetup._validators)))
+@docstring.Substitution(
+    "\n".join(
+        sorted(
```

Comment:
Is it desirable to sort the list of valid rcParams alphabetically or should I remove the sorting? (Same for `STYLE_BLACKLIST`)

Reply:
I am weakly 👍🏻 on sorting.

Reply:
`rcsetup._validators` is a dictionary, so it should be ordered. But that being said, I don't think much thought went into the order, so it's not a very good one. The only bit of ordering really is that `_internal.classic_mode` comes last, and there's some grouping, but this fortunately mostly corresponds with common prefixes, and would be preserved by sorting. So I guess that's sort of a 👍🏾 for sorting.

But seeing `_internal.classic_mode` there, I think this list should not include any rcParam with a `_` prefix.

Reply:
> there's some grouping, but this fortunately mostly corresponds with common prefixes, and would be preserved by sorting

Yes, but rcParams related to backends will be split (e.g. `backend` and `interactive` or the `webagg.*` rcParams).